.docker/database/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM mariadb:latest
```

Comment:
I'd definitely use a specific version that is on prod. That's currently mysql-5.5, but find the oldest version you can and we'll keep them in sync once we upgrade :)

If there is a way to keep it in sync with the terraform version, that would be golden :)

Reply:
I added version 10.0 that should be more or less compatible with mysql-5.5. For local developing mysql 5.5 and mariadb 5.5 do not have official docker images for arm64v8, do they? At least for it is this error:

<img width="756" alt="image" src="https://github.com/dracidoupe/graveyard/assets/21171948/0cac3fab-1089-45d8-99e7-cbcce51bf0f3">